pywbem/_subscription_manager.py

```diff
@@ -541,6 +557,22 @@ def add_listener_destinations(self, server_id, listener_urls, owned=True):
             permanent. See :ref:`WBEMSubscriptionManager` for details about
             these ownership types.
 
+          persistence_type (:class:`py:int` or None):
```

Comment:
Can we not have string values "permanent" and "transient" instead of the integer values?

Reply:
Rewrote the validate method to use strings and validate them.  The _create_listenerdestination still uses integers as converted in add_listener_destination.

DONE

Reply:
The latest version of the code does not have such a change, but given that we stick with the integer in the `add_listener_destinations()` method, I don't see why we would change the internal validate method to use strings.
Ok as it is now.